Documentation/configuration.md

```diff
@@ -3,7 +3,7 @@
 ## Automatic
 
 You can use an USB drive with HassOS to configure network options, SSH access to the host and to install updates.
-Format a USB stick with FAT32/EXT4/NTFS and name it `CONFIG`. Use the following directory structure within the USB drive:
+Format a USB stick with FAT32/EXT4/NTFS and name it `CONFIG`. Alternative you can create a `CONFIG` folder inside boot partition. Use the following directory structure within the USB drive:
```

Comment:
You don't seem to indicate which partition the is the boot, a default install only has:
- disk
- hassos-data
- hassos-kernel
- hassos-overlay

Are we to assume the full path is:
  /hassos-overlay/boot/CONFIG

Or another location? I've tried to read the code here, but don't fully understand who /mnt/boot/CONFIG will be created after the rasberry pi is full booted

Reply:
If I understand correctly, the 'CONFIG' folder should be create at the root of the disk partition.
It will then be mounted in /mnt/config/CONFIG
If someone try, please open a PR to clarify the documentation. Thank you.